app/models/messages/Consignment.scala

```diff
@@ -25,7 +25,9 @@ case class Consignment(
   LocationOfGoods: Option[LocationOfGoods],
   ActiveBorderTransportMeans: Option[List[ActiveBorderTransportMeans]],
```

Comment:
May be unrelated to the PR, but do we need an Option[List]] here?

Reply:
I'm not a fan of Option[List[T]] either but we'd have to update the reads if we changed it to List[T] to do a .readWithDefault(Nil) otherwise we'll fail to read the payload for an undefined Option